BORRADO USUARIOS...
RESULTADO REMOVE: true
********************** TEST 1 (value: 1)**********************
El admin se logea, cambia la contraseña, log out y login con la nueva
catch RemoteException with message: Conexión rehusada (Connection refused)
FAILED
********************** TEST 2 (value: 1)**********************
El admin se logea, añade dos usuarios iguales, el segundo debe fallar, y luego logout
catch RemoteException with message: Conexión rehusada (Connection refused)
FAILED
********************** TEST 6 (value: 1)**********************
user2 login (falla), admin login, crea user2, user2 login otro st, remove user1 (falla), admin borra user1 (ok) borra user que no existe (falla)
catch RemoteException with message: Conexión rehusada (Connection refused)
FAILED
********************** TEST 8 (value: 1)**********************
user2 login st1 y st2, st1 logout, changpwd en st1 (fail), changpwd en st2
catch RemoteException with message: Conexión rehusada (Connection refused)
FAILED
********************** TEST 10 (value: 1)**********************
user2 login 2 veces en st1, logout, admin login st1 (fail), logout, admin login st1 (ok)
catch RemoteException with message: Conexión rehusada (Connection refused)
FAILED
********************** TEST 11 (value: 1)**********************
user2 login st1, admin login st2, admin borrar user2, st1 hacer login user2 (fail)
catch RemoteException with message: Conexión rehusada (Connection refused)
FAILED
********************** TEST 14 (value: 1)**********************
admin crea usuer11, user11 se logea en dos stubs, st1 y st2, en uno se borra y en otro no debe permitir hacer nada.
catch RemoteException with message: Conexión rehusada (Connection refused)
FAILED
********************** TEST 15 (value: 1)**********************
admin crea user4, user4 addF user 5 en st2 (fail), user4 login en st2, user4 addFol user5 (fail), admin crear user5, use4 añade al user5 (ok)
catch RemoteException with message: Conexión rehusada (Connection refused)
FAILED
********************** TEST 16 (value: 1)**********************
user4 log en st1 y getFollower, ok y user5. user5 logea en st2 y getfollowers, ok y nada.
catch RemoteException with message: Conexión rehusada (Connection refused)
FAILED
********************** TEST 18 (value: 1)**********************
admin login, admin addW vino 1; ok, getWines; ok y vino1. addW vino2, getWines; ok y vino2-vino1	vinos en un stub
catch RemoteException with message: Conexión rehusada (Connection refused)
FAILED
